### PR TITLE
fix missing stack property

### PR DESCRIPTION
**Pull request by aduh95**
Refs: https://github.com/nodejs/node/pull/52189#issuecomment-2016197258

### CODE REVIEW

File: test/output/test-nested.xml [hunk 1]
```diff
@@ -5,24 +5,12 @@
...
----
-code: ERR_TEST_FAILURE
-duration_ms: 2.332323873
-error: Promise resolution is still pending but the event loop has already resolved
-failureType: cancelledByParent
-...
-</failure>
+			<failure type="failure" message=" (0)">process.emit (node:events:513:28)
+&lt;anonymous&gt; (node:events:51:28)
+&lt;anonymous&gt; (node:events:512:28)</failure>
```
Comment (richardlau) [inline test/output/test-nested.xml:10]: Do we still want to include the "error" in addition to the "stack"?

**Review by MoLow — APPROVED** [review]

**Comment by cclauss** [comment]
@MoLow @richardlau How should we fix the failing tests?  Which is correct -- the actual vs. the expected?

**Comment by MoLow** [comment]
Not sure. CC @aduh95

**Comment by aduh95** [comment]
i have no idea, the tests pass on my machine. It looks a rounding error 🤔

**Review by MoLow — APPROVED** [review]

**Comment by aduh95** [comment]
OK, it looks like it's a difference between Python 3.9 (that I was using locally) and Python 3.12 (that the CI was using). Generating the fixtures with Python 3.12 seems to solve the issue.

**Comment by cclauss** [comment]
Great!  The only remaining unanswered question is https://github.com/nodejs/tap2junit/pull/54#discussion_r1536677768

**Review by richardlau — APPROVED** [review]

**Comment by MoLow** [comment]
@cclauss can you release this to Pypi?

**Comment by aduh95** [comment]
Ping @nodejs/python, any chance we could have this patch in a release so the CI starts using it?

**Pull request merged**